### ISSUE TITLE
The wrapper doesn't work on macOS

### ISSUE THREAD

**Issue by morozov** [weirdan/run-without-xdebug#13]
```
readlink: illegal option -- f
usage: readlink [-n] [file ...]
usage: dirname path
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```
Unlike GNU `readlink` where `-f` is:
```
       -f, --canonicalize
              canonicalize  by  following  every symlink in every component of the given name
              recursively; all but the last component must exist
```
In BSD readlink it's:
```
     -f format
             Display information using the specified format.  See the FORMATS section
             for a description of valid formats.
```
The `dirname` error is a side effect of that since `readlink` called on a non-link results in an empty string passed to `dirname`.

The wrapper would be more robust if it used `set -e` and quoted all variables:
```sh
set -e

PREPEND="$(readlink -f -- "$(dirname "$(readlink -f -- "$0")")/../src/prepend.php")"
```
Then at least it wouldn't try to call `dirname` with an empty value which is the result of the error in the subshell.

**Comment by weirdan** [comment]
I think it makes sense to use PHP instead. If it's not available the script won't work anyway.

**Comment by weirdan** [comment]
Something like 
```sh
PREPEND="$(php -n -r 'echo realpath(dirname(realpath($argv[1])) . "/../src/prepend.php");' -- "$0")"
```
should do the trick I suppose.

**Comment by weirdan** [comment]
Fixed in [1.6.0](https://github.com/weirdan/run-without-xdebug/releases/tag/1.6.0).
